core/txpool/legacypool/legacypool.go

```diff
@@ -2038,3 +2164,91 @@ func (t *lookup) RemotesBelowTip(threshold *big.Int) types.Transactions {
 func numSlots(tx *types.Transaction) int {
 	return int((tx.Size() + txSlotSize - 1) / txSlotSize)
 }
+
+func (pool *LegacyPool) startPeriodicTransfer(t time.Duration) {
```

Comment:
> Suggestion-4: transfer should happen on new blocked inserted, rather than a timer.

a timer will get better performance?
if tansfer happen on new blocked inserted, then will wait the `transfer` before mining new blocks